Capri.Services/PromoterService.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+using Capri.Database;
+using Capri.Database.Entities;
+using Capri.Database.Entities.Identity;
+using Capri.Web.ViewModels.Promoter;
+
+namespace Capri.Services
+{
+    public class PromoterService : IPromoterService
+    {
+        private readonly ISqlDbContext _context;
+
+        public PromoterService(ISqlDbContext context)
+        {
+            _context = context;
+        }
+
+        public IServiceResult<Promoter> Get(Guid id)
+        {
+            Promoter promoter = _context.Promoters.Find(id);
```

Comment:
Use `var` keyword. I know that Java doesn't have anything like this, but C# has ;) 
Other thing, if I'm not mistaken `Find` can cause you exception and is synchronous operation to database. Better way would be using `FirstOrDefaultAsync`. Read about that.